### PR TITLE
BL-1055-resolve-swagger-issues-in-codebase

### PR DESCRIPTION
**Pull request by JordonP123**
# Description
The console was rendering errors related to swagger.  I went through and updated the syntax errors, mainly occurring in the endpoints at the beginning.  I ran into some issues trying to format the YAML, reached out to Jake, and Jake went in and formatted the YAML correctly.  The objects were written in JSON format, and were not being referenced correctly.  Jake switched the format of the objects to YAML, and referenced the YAML as necessary.

### CODE REVIEW

**Comment by JordonP123** [comment]
https://www.loom.com/share/589e8afdf0da4b438dc3382090de99ef

**Review by ashtilawat — DISMISSED** [review]

File: api/client/clientRouter.js [hunk 1]
```diff
@@ -80,10 +80,12 @@ router.put(':id', (req, res) => {
 
 /**
  * @swagger
- * /:id/notes
+ * /{id}/notes:
  *  get:
```
Comment (ashtilawat) [inline api/client/clientRouter.js:84]: What did you switch from using a `:` to using curly brackets? 
Did this have to do with the JSON formatting correctly?

File: api/client/clientRouter.js [hunk 2]
```diff
@@ -230,9 +239,11 @@ router.put('/:clientID/notes/:noteID', async (req, res) => {
 
 /**
  * @swagger
- * /:id/notes/:id
+ * /{id}/notes/{id}:
  *  put:
```
Comment (ashtilawat) [inline api/client/clientRouter.js:243]: Test comment for Ash's video on how to reviews PRs

File: data/seeds/18-contact_preferences.js [hunk 3]
```diff
@@ -20,4 +20,4 @@ exports.seed = function (knex) {
         },
       ]);
     });
-};
```
Comment (ashtilawat) [inline data/seeds/18-contact_preferences.js]: Looks like there is an eslint issue here. I think we just need to add a blank line at the end of this code.

**Review by ashtilawat — CHANGES_REQUESTED** [review]
I have changes for this PR I wanted to run through: 

- curly brackets over `:`?
- eslint error at the end of files

File: api/client/clientRouter.js (hunk 1 above)
Comment (JordonP123) [reply api/client/clientRouter.js:84]: Yes, this is the correct way to format YAML.

**Review by ashtilawat — DISMISSED** [review]

File: .eslintrc.js [hunk 4]
```diff
@@ -9,7 +9,14 @@ module.exports = {
     ecmaVersion: 11,
   },
   plugins: ['prettier'],
-  rules: {},
+  rules: {
```
Comment (ramib5734) [inline .eslintrc.js:12]: What did adding objects to 'rules' do to help with the swagger issues?

File: .prettierrc.js [hunk 5]
```diff
@@ -3,4 +3,5 @@ module.exports = {
   tabWidth: 2,
   semi: true,
   singleQuote: true,
+  endOfLine: 'auto',
```
Comment (ramib5734) [inline .prettierrc.js:6]: What is the purpose of this `endOfLine: 'auto',` line?

**Review by ramib5734 — COMMENTED** [review]
Just looking through the code and was wondering what those lines of code you added that I commented on did to change the outcome of the swagger issues, if any!

**Pull request merged**